lua/astrocommunity/lsp/coc-nvim/init.lua

```diff
@@ -40,6 +40,7 @@ return {
         maps.n["<Leader>lR"] = maps.n.gr
         maps.n["<Leader>lr"] = { "<Plug>(coc-rename)", desc = "Rename current symbol" }
         maps.n["<Leader>lf"] = { function() vim.cmd.Format() end, desc = "Format buffer", nowait = true }
+        maps.x["<Leader>l;"] = { "<Plug>(coc-format-selected)", desc = "Format selection", nowait = true }
```

Comment:
```suggestion
        maps.x["<Leader>lF"] = { "<Plug>(coc-format-selected)", desc = "Format selection", nowait = true }
```

Maybe this makes more sense?